txpipe/diagnostic_maps.py

```diff
@@ -60,6 +62,7 @@ class TXDiagnosticMaps(PipelineStage):
         'true_shear' : False,
         'flag_exponent_max': 8,
         'dilate': True,
+        'psf_prefix': 'psf_',
```

Comment:
are you imagining there to be flexibility here? if we are compiling the input shear catalog can't we always call it a fix name?

Reply:
Yep, exactly, as you mention below.  It's mainly because mcal_psf is a rounded version which isn't that useful for diagnostics.